src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -82,6 +82,27 @@
     </PropertyGroup>
   </Target>
 
+  <Target Name="AddCrossAssemblyOptimizationSwitch"
+    Condition="'$(TargetFramework)' != 'net6.0'"
```

Comment:
shouldnt be necessary, this assembly is .net9 only

Reply:
ah, tx

Reply:
```suggestion
```